src/Project/functions/createNodeModulesPathMapping.ts

```diff
@@ -1,15 +1,18 @@
+import ts from "byots";
 import fs from "fs-extra";
 import path from "path";
 import { realPathExistsSync } from "Shared/util/realPathExistsSync";
 
 export function createNodeModulesPathMapping(typeRoots: Array<string>) {
+	const getCanonicalFileName = ts.createGetCanonicalFileName(ts.sys.useCaseSensitiveFileNames);
+
 	const nodeModulesPathMapping = new Map<string, string>();
 	// go through each org
 	for (const scopePath of typeRoots) {
 		if (fs.pathExistsSync(scopePath)) {
 			// map module paths
 			for (const pkgName of fs.readdirSync(scopePath)) {
-				const pkgPath = path.join(scopePath, pkgName);
+				const pkgPath = getCanonicalFileName(path.join(scopePath, pkgName));
```

Comment:
Maybe instead of putting it here, it should go on both sides of the .set() call?

Reply:
Sure, it's now in ca2b4984864d52c172604a7b1c6823659f164837